libs/backend/src/scan/cast_vote_records/legacy_export.test.ts

```diff
@@ -32,6 +29,11 @@ import {
   ResultSheet,
 } from './legacy_export';
 
+jest.mock('@votingworks/auth', (): typeof import('@votingworks/auth') => ({
+  ...jest.requireActual('@votingworks/auth'),
+  writeSignatureFile: jest.fn(),
```

Comment:
I've removed mocking of the writing of signature files in most places. I just couldn't in this test file since all data writing is itself mocked, so `writeSignatureFile` can't read in the data that it needs to sign. In other test files, where we actually write data to the file system, `writeSignatureFile` can access the data that it needs to sign and works just fine.

Didn't feel worth updating since I'll soon be replacing this test file anyway.